packages/trader/src/Modules/Trading/Components/Form/TradeParams/__tests__/amount-mobile.spec.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mockStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types';
+import { TURBOS } from '@deriv/shared';
+import { useTraderStore } from 'Stores/useTraderStores';
+import TraderProviders from '../../../../../../trader-providers';
+import Amount from '../amount-mobile';
+
+const mockedNumpadComponent = 'Mocked Numpad Component';
+const default_props = {
+    toggleModal: jest.fn(),
+    duration_value: 3,
+    duration_unit: 'm',
+    has_duration_error: false,
+    amount_tab_idx: undefined,
+    setAmountError: jest.fn(),
+    setAmountTabIdx: jest.fn(),
+    setSelectedAmount: jest.fn(),
+    stake_value: 10,
+    payout_value: 20,
+};
+const default_mock_store = {
+    modules: {
+        trade: {
+            basis: 'stake',
+            basis_list: [{ text: 'Payout', value: 'payout' }],
+            contract_type: 'rise_fall',
+            is_turbos: false,
+            is_vanilla: false,
+            onChangeMultiple: jest.fn(),
+            amount: 10,
+            duration_unit: 'm',
+            duration: 3,
+            stake_boundary: { [TURBOS.LONG.toUpperCase()]: { min_stake: 1, max_stake: 10000 } } as ReturnType<
```

Comment:
Why does it have to be in uppercase?

Reply:
Because in real `stake_boundary` it is written in uppercase. For e.g. on line 56 _amount-mobile.tsx_ file upperCase is applied to `contract_type` in order to find match with keys in `stake_boundary`. So, if I wouldn't apply it in my test file, there will be no match.
`const { min_stake, max_stake } = stake_boundary[contract_type.toUpperCase()] || {};`